src/_P103_Atlas_EZO_pH_ORP_EC.ino

```diff
@@ -128,11 +127,9 @@ boolean Plugin_103(byte function, struct EventStruct *event, String& string)
       if (status) {
         String boardStatus(statussensordata);
 
-        addHtml(F("<TR><TD>Board restart code: </TD><TD>"));
-        int pos1 = boardStatus.indexOf(',');
-        int pos2 = boardStatus.lastIndexOf(',');
+        addRowLabel(F("Board restart code"));
 
-        switch ((char)boardStatus.substring(pos1 + 1, pos2)[0])
+        switch ((char)boardStatus.substring(boardStatus.indexOf(',') + 1, boardStatus.lastIndexOf(','))[0])
```

Comment:
This is rather tricky.
What if the indexOf does return -1?
Also you're casting a `String` object to a char. No idea what it will return if the substring is an empty string.

Reply:
fixed